server/services/Parser.js

```diff
@@ -1,8 +1,45 @@
-const {MealTimes, Dates} = require('../models/Identifier');
-const StopWord = require('../models/StopWord');
+const { mealTimes, datesLog } = require("../models/Identifier");
+const { stopWordList } = require("../models/StopWord");
 
-class Parser {
-    //TODO Parse through and identify all StopWords & Identifiers
+function parser(string) {
```

Comment:
- We should not name a function parameter `string` 
- string is usually a type in other languages, we should use `words` or other meaningful plurals